data/landSales/LandPreSale_5/prices.ts

```diff
@@ -0,0 +1,18 @@
+import {BigNumber} from '@ethersproject/bignumber';
+
+function sandWei(v: number) {
+  return BigNumber.from(v).mul("1000000000000000000").toString();
+}
+
+// These are Mainnet prices based on 0.036367$ per SAND
```

Comment:
Are these the latest prices?

Reply:
they are the price we fixed once.
They might changes. will need to update once they are known. we also have the multiplier setup so we can do just at the time the presale start